src/app/components/HoldConfirmation/HoldConfirmation.jsx

```diff
@@ -32,6 +32,11 @@ class HoldConfirmation extends React.Component {
    */
   requireUser() {
     if (this.state.patron && this.state.patron.id) {
+      const queryID = this.props.location.query ? this.props.location.query.id : '';
```

Comment:
Yeah, I don't think this will actually be useful because we are using the patron id. Serverside is preferred. We have to think on this a lot more.